networkx/classes/function.py

```diff
@@ -54,7 +54,15 @@
 
 
 def nodes(G):
-    """Returns an iterator over the graph nodes."""
+    """Returns an iterator over the graph nodes.
+
+    Examples
+    --------
+    >>> G = nx.Graph()
+    >>> G.add_nodes_from([1, 2, 3, 4, 5])
```

Comment:
This is just a nit, but it feels a little circular to explicitly create a graph with a node list then immediately reproduce the nodes! This is of course correct, but I think you'd get the same effect with using a well-known graph constructor, e.g.

```suggestion
    >>> G = nx.path_graph(5)
```